perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -3974,6 +3975,25 @@ public Void call(ApiCaller ac, Deserializer parms) throws AttributeAlreadyMarked
 		}
 	},
 
+	/*#
+	 * Converts attribute to nonunique - unmarks unique flag from attribute definition, and deletes all values
+	 * from a special table with unique constraint that ensures that all values remain unique.
+	 * Already nonunique attribute is skipped.
```

Comment:
I don't think that ``Already nonunique attribute is skipped.`` sentence is true. I think, that exception is thrown instead.

Reply:
Removed.